distribution/std-lib/Standard/src/Base/Data/Vector.enso

```diff
@@ -119,7 +117,8 @@ type Vector
 
        > Example
          Checking the length of a vector.
-             [1, 2, 3, 4].length == 4
+
+             [1, 2, 3, 4].length
```

Comment:
AFAIK, for the `[ ]` syntax we need a `Standard.Base` import.

Here and below.

Reply:
The assumption for all examples is that `from Standard.Base import all` is present. I should document this.

Reply:
I've now documented this.